contents/docs/api/user.mdx

```diff
@@ -96,4 +96,4 @@ Example response:
    },
    "posthog_version":"1.6.0"
 }
-```
\ No newline at end of file
+```
```

Comment:
Think it's worth updating the sample response as this has completely changed.

Reply:
what are your thoughts on just removing that? I feel like it would get out of date soon again

Reply:
Well I think it can actually be quite useful for users, even documenting some non-obvious elements could potentially be helpful. We should in theory only add attributes and not remove or edit, so I think it should be fine to keep. Might be worth keeping a note in the code base to update this doc when we change it.

Reply:
Since we currently are just providing the json output, which is super easy to get via `curl` or going to the UI. It didn't seem all that useful to our users and just causes us more work to keep up to date.

Reply:
Makes a lot of sense!